app/javascript/components/public_health/actions/UpdateCaseStatus.js

```diff
@@ -6,21 +6,29 @@ import axios from 'axios';
 
 import reportError from '../../util/ReportError';
```

Comment:
When moving to the isolation workflow with the case status bulk action I'm seeing the history providing a blank reason (see screenshot). I'm thinking the "reason" text should be omitted altogether here.

<img width="1502" alt="Screen Shot 2021-05-27 at 11 57 18 AM" src="https://user-images.githubusercontent.com/24628687/119874317-b300ea00-bee2-11eb-8f4a-a2b2f81bfeb6.png">

Reply:
It looks like this is also the case when changing the case status from the patient show page

Reply:
I think this is due to appending a '.' to the reason if it already does not end with one. I'm thinking we could just check if the reasoning is empty before adding on the '.'

Reply:
And the same issue happens when closing a record via the close bulk action without specifying any reasoning.

Reply:
It's supposed to be! Hmm i'll take a look!

Reply:
I'll also flag in this thread since it's related - I'm seeing a similar issue when simply doing a Bulk Case Status Update in general. This history entry resulted from updating the case status for an Isolation monitoree from `Probable` to `Confirmed`. The modal never provides the opportunity to set a Reason so it shouldn't be displayed in the history entry. This is an issue for pretty much all Case Status bulk update actions that don't specifically provide the user with an option to input a note.

![image](https://user-images.githubusercontent.com/42656458/119996305-9b863780-bf9c-11eb-9c1b-f3e92eb052c5.png)

Reply:
Nice catch! This was a result of my trying to be fancy and always appending a period to the end without checking if it even exitsed. Fixed.